web_widget_radio_tree/static/src/js/widget.js

```diff
@@ -0,0 +1,90 @@
+openerp.web_widget_radio_tree = function (instance) {
+
+    var QWeb = instance.web.qweb;
+
+    instance.web.list.columns.add('field.radio_tree', 'instance.web.list.RadioTreeColumn');
+
+    instance.web.list.RadioTreeColumn = instance.web.list.Column.extend({
+        _format: function (row_data, options) {
+            var name = options.model + '_' + this.id;
```

Comment:
why not letting qweb taking care of the formatting and passing the whole options object to the template? And row_data too just for good measure.